stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -1,598 +1,1356 @@
-      },
-      "privateIpAddress": {
-        "key": "ipv4-addr.value"
+    "natSourceAddress": [
+      {
+        "key": "ipv4-addr.value",
+        "object": "nat_src_ip"
       },
-      "publicIpAddress": {
-        "key": "ipv4-addr.value"
+      {
+        "key": "network-traffic.x_nat_src_ref",
+        "object": "nt",
+        "references": "nat_src_ip"
+      }
+    ],
+    "natSourcePort": {
+      "key": "network-traffic.x_nat_source_port",
+      "object": "nt",
+      "transformer": "ToInteger"
+    },
+    "protocol": {
+      "key": "network-traffic.protocols",
+      "object": "nt",
+      "group": true,
+      "transformer": "ToLowercaseArray"
+    },
+    "riskScore": {
+      "key": "network-traffic.x_riskScore",
+      "object": "nt"
+    },
+    "sourceAddress": [
+      {
+        "key": "ipv4-addr.value",
+        "object": "src_ip"
       },
-      "riskScore": {
-        "key": "x-msazure-sentinel-alert.hostStates.riskScore",
-        "object": "alert"
+      {
+        "key": "network-traffic.src_ref",
+        "object": "nt",
+        "references": "src_ip"
       }
+    ],
+    "sourceLocation": {
+      "key": "x-ibm-finding.src_geolocation",
+      "object": "finding"
     },
-    "id": {
-      "key": "x-msazure-sentinel-alert.providerid",
-      "object": "alert"
+    "sourcePort": {
+      "key": "network-traffic.src_port",
+      "object": "nt",
+      "transformer": "ToInteger"
     },
-    "incidentIds": {
-      "key": "x-msazure-sentinel-alert.incidentIds",
-      "object": "alert",
-      "transformer": "ToString"
+    "status": {
+      "key": "network-traffic.x_status",
+      "object": "nt"
     },
-    "lastModifiedDateTime": [
+    "urlParameters": {
+      "key": "network-traffic.x_url_parameters",
+      "object": "nt"
+    }
+  },
+  "processes": {
+    "accountName": [
       {
-        "key": "modified",
-        "cybox": false
+        "key": "user-account.user_id",
+        "object": "user"
       },
       {
-        "key": "x-msazure-sentinel-alert.lastmodifieddatetime",
-        "object": "alert"
+        "key": "process.creator_user_ref",
+        "object": "process",
+        "references": "user"
       }
     ],
-    "malwareStates": {
-      "category": {
-        "key": "x-msazure-sentinel-alert.malwareStates.category",
-        "object": "alert"
-      },
-      "family": {
-        "key": "x-msazure-sentinel-alert.malwareStates.family",
-        "object": "alert"
-      },
-      "name": {
-        "key": "x-msazure-sentinel-alert.malwareStates.name",
-        "object": "alert"
-      },
-      "severity": {
-        "key": "x-msazure-sentinel-alert.malwareStates.severity",
-        "object": "alert"
-      },
-      "wasRunning": {
-        "key": "x-msazure-sentinel-alert.malwareStates.wasRunning",
-        "object": "alert"
+    "commandLine": {
+      "key": "process.command_line",
+      "object": "process"
+    },
+    "createdDateTime": {
+      "key": "process.created",
```

Comment:
Should be `process.created_time` for 2.1

Reply:
fixed